app/views/finders/_results.mustache

```diff
@@ -39,4 +39,12 @@
   {{/documents}}
 </ul>
 
+{{#zero_results}}
+  <div class='no-results'>
+    <h2 class='gem-c-heading'>There are no matching results.</h2>
+    <p>Try making your search broader.</p>
+    {{results}}
```

Comment:
What is the intent of `{{results}}` here?

Reply:
good catch, that's my mistake, will remove.